src/assets/icons/diamond.svg

```diff
@@ -0,0 +1 @@
5v1.523h1.305v-1.522z" fill="#8ED4E6"/><path d="M22.171 10.65H20.87v1.523h1.302v-1.522zM20.87 10.65h-1.306v1.523h1.305v-1.522zM19.562 10.65h-1.303v1.523h1.303v-1.522z" fill="#8ED4E6"/><path d="M18.26 10.65h-1.303v1.523h1.303v-1.522zM16.957 10.65h-1.305v1.523h1.305v-1.522z" fill="#8ED4E6"/><path d="M15.65 10.65h-1.302v1.523h1.302v-1.522z" fill="#BAE4EB"/><path d="M14.348 10.65h-1.305v1.523h1.305v-1.522zM13.04 10.65h-1.302v1.523h1.302v-1.522zM11.738 10.65h-1.302v1.523h1.302v-1.522zM10.436 10.65H9.13v1.523h1.305v-1.522zM9.128 10.65H7.826v1.523h1.302v-1.522zM7.826 10.65H6.521v1.523h1.305v-1.522zM6.519 10.65H5.217v1.523h1.302v-1.522z" fill="#67CBDF"/><path d="M23.476 9.128h-1.305v1.523h1.305V9.128z" fill="#8ED4E6"/><path d="M22.171 9.128H20.87v1.523h1.302V9.128zM20.87 9.128h-1.306v1.523h1.305V9.128zM19.562 9.128h-1.303v1.523h1.303V9.128z" fill="#8ED4E6"/><path d="M18.26 9.128h-1.303v1.523h1.303V9.128zM16.957 9.128h-1.305v1.523h1.305V9.128z" fill="#8ED4E6"/><path d="M15.65 9.128h-1.302v1.523h1.302V9.128zM14.348 9.128h-1.305v1.523h1.305V9.128zM13.04 9.128h-1.302v1.523h1.302V9.128zM11.738 9.128h-1.302v1.523h1.302V9.128zM10.436 9.128H9.13v1.523h1.305V9.128zM9.128 9.128H7.826v1.523h1.302V9.128zM7.826 9.128H6.521v1.523h1.305V9.128z" fill="#67CBDF"/><path d="M22.171 7.609H20.87v1.522h1.302V7.61zM20.87 7.609h-1.306v1.522h1.305V7.61zM19.562 7.609h-1.303v1.522h1.303V7.61z" fill="#8ED4E6"/><path d="M18.26 7.609h-1.303v1.522h1.303V7.61zM16.957 7.609h-1.305v1.522h1.305V7.61z" fill="#8ED4E6"/><path d="M15.65 7.609h-1.302v1.522h1.302V7.61zM14.348 7.609h-1.305v1.522h1.305V7.61zM13.04 7.609h-1.302v1.522h1.302V7.61zM11.738 7.609h-1.302v1.522h1.302V7.61zM10.436 7.609H9.13v1.522h1.305V7.61zM9.128 7.609H7.826v1.522h1.302V7.61z" fill="#67CBDF"/><path d="M20.87 6.087h-1.306v1.522h1.305V6.087zM19.562 6.087h-1.303v1.522h1.303V6.087z" fill="#8ED4E6"/><path d="M18.26 6.087h-1.303v1.522h1.303V6.087zM16.957 6.087h-1.305v1.522h1.305V6.087z" fill="#8ED4E6"/><path d="M15.65 6.087h-1.302v1.522h1.302V6.087zM14.348 6.087h-1.305v1.522h1.305V6.087zM13.04 6.087h-1.302v1.522h1.302V6.087zM11.738 6.087h-1.302v1.522h1.302V6.087zM10.436 6.087H9.13v1.522h1.305V6.087z" fill="#67CBDF"/><path d="M19.562 4.564h-1.303v1.523h1.303V4.564z" fill="#8ED4E6"/><path d="M18.26 4.564h-1.303v1.523h1.303V4.564zM16.957 4.564h-1.305v1.523h1.305V4.564z" fill="#8ED4E6"/><path d="M15.65 4.564h-1.302v1.523h1.302V4.564zM14.348 4.564h-1.305v1.523h1.305V4.564zM13.04 4.564h-1.302v1.523h1.302V4.564zM11.738 4.564h-1.302v1.523h1.302V4.564z" fill="#67CBDF"/><path d="M18.26 3.042h-1.303v1.522h1.303V3.042zM16.957 3.042h-1.305v1.522h1.305V3.042z" fill="#8ED4E6"/><path d="M15.65 3.042h-1.302v1.522h1.302V3.042zM14.348 3.042h-1.305v1.522h1.305V3.042zM13.04 3.042h-1.302v1.522h1.302V3.042z" fill="#67CBDF"/><path d="M16.957 1.52h-1.305v1.522h1.305V1.519z" fill="#8ED4E6"/><path d="M15.65 1.52h-1.302v1.522h1.302V1.519zM14.348 1.52h-1.305v1.522h1.305V1.519z" fill="#67CBDF"/><path d="M15.65 0h-1.302v1.52h1.302V0z" fill="#67CBDF"/></svg>
```

Comment:
Are these optimized? :)

Reply:
Yup, should be! All SVGs have been run though `svgo`'s optimizer. Or is there a different way they should be optimized?

Reply:
Nope, was just checking haha